src/controllers/index.controller.js

```diff
@@ -222,9 +223,35 @@ function multisigPubkey(req, res) {
   });
 }
 
+function getConfigurationString(req, res) {
+  const validationResult = parametersValidation(req);
```

Comment:
I thought it slightly confusing that the execution of this validation is separated from the definition of what's being validated., without even a reference. I had to dig for a while to find out where we define what to validate (in the `index.routes.js`)

But I believe this is something broader than the scope of this PR, since the other routes are probably using the same pattern.

Reply:
I also this this is strange but it's how the `express-validator` works.
I don't think this is an issue. It's strange but it's the more generic approach, the developer can choose how this error is treated (return 200 with error, 400, noop, log, error service, etc)